tests/foreman/cli/test_host.py

```diff
@@ -237,139 +259,20 @@ def test_positive_add_interface_by_id(self):
-            interface has correct domain name
-
-        :BZ: 1384497
-
-        :CaseLevel: Integration
-
-        :CaseImportance: Critical
-        """
-        name = gen_string('alpha').lower()
-        hostgroup = make_hostgroup({
-            'content-view-id': self.new_cv['id'],
-            'lifecycle-environment-id': self.new_lce['id'],
-            'organization-ids': self.new_org['id'],
-        })
-        host = entities.Host(
-            organization=entities.Organization(id=self.new_org['id']).read()
-        )
-        host.create_missing()
-        interface = (
-            "type=interface,mac={0},identifier=eth0,name={1},domain={2},"
-            "ip={3},primary=true,provision=true"
-        ).format(host.mac, gen_string('alpha'), host.domain.name, gen_ipaddr())
-        result = make_host({
-            u'architecture-id': host.architecture.id,
-            u'hostgroup-id': hostgroup['id'],
-            u'location-id': host.location.id,
-            u'medium-id': host.medium.id,
-            u'name': name,
-            u'operatingsystem-id': host.operatingsystem.id,
-            u'organization-id': host.organization.id,
-            u'partition-table-id': host.ptable.id,
-            u'root-password': host.root_pass,
-            u'interface': interface,
-        })
-        self.assertEqual(
-            '{0}.{1}'.format(name, host.domain.read().name),
-            result['name'],
-        )
-        host_interface = HostInterface.info({
-            u'host-id': result['id'],
-            u'id': result['network-interfaces'][0]['id']
-        })
-        self.assertEqual(host_interface['domain'], host.domain.read().name)
-
-    @tier1
-    def test_positive_create_with_org_name(self):
-        """Check if host can be created with organization name
-
-        :id: c08b0dac-9820-4261-bb0b-8a78f5c78a74
-
-        :expectedresults: Host is created using organization name
-
-        :CaseImportance: Critical
-        """
-        new_host = make_fake_host({
-            'content-view-id': self.DEFAULT_CV['id'],
-            'lifecycle-environment-id': self.LIBRARY['id'],
-            'organization': self.new_org['name'],
-        })
-        self.assertEqual(new_host['organization'], self.new_org['name'])
-
-    @skip_if_bug_open('bugzilla', 1483252)
-    @tier2
-    def test_positive_create_with_content_source(self):
-        """Create a host with content source specified
+    def test_positive_create_and_update_with_content_source(self):
+        """Create a host with content source specified and update content
+            source
 
-        :id: 6068bd4d-18d8-47a2-99f4-3e0ee9208104
+        :id: 5712f4db-3610-447d-b1da-0fe461577d59
 
         :customerscenario: true
 
-        :BZ: 1260697, 1483252, 1313056
+        :BZ: 1260697, 1483252, 1313056, 1488465
 
         :expectedresults: A host is created with expected content source
-            assigned
+            assigned and updated content source
```

Comment:
This is a bit confusing,  - you already expected content source assigned and some another content source update 
It's not that clear. 
I think better is:
`A host is created with expected content source assigned and then content source is successfully updated`